test/Atmos/EDMF/ekman_layer.jl

```diff
@@ -27,6 +27,7 @@ include("edmf_model.jl")
 include("edmf_kernels.jl")
 
 CLIMAParameters.Planet.T_surf_ref(::EarthParameterSet) = 290.0
+CLIMAParameters.Atmos.EDMF.a_surf(::EarthParameterSet) = 0.0
```

Comment:
Does this work off-the-branch? `a_min::FT = 0.001` in edmf_model.jl

Reply:
To run the ekman_layer.jl with EDMF now I am commenting out all tendencies. If I do that it can run with 
`CLIMAParameters.Atmos.EDMF.a_surf(::EarthParameterSet) = 0.0` and `a_min::FT = 0.001`

Reply:
Ok. And have you looked at the profiles? Is a_surf=0 or a_surf=a_min ?

Reply:
Printing in the EDMF BC 'a_surf=0', but the profile is set somewhere in the code to 'a_min' so that the output profile is 'a=a_min'